src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -90,4 +102,18 @@ public ResponseEntity<String> noRelationStationExceptionHandler(
         return ResponseEntity.badRequest()
             .body(exception.getMessage());
     }
+
+    @ExceptionHandler(SingleSectionException.class)
+    public ResponseEntity<String> singleSectionExceptionHandler(
+        final SingleSectionException exception) {
+        return ResponseEntity.badRequest()
+            .body(exception.getMessage());
+    }
+
+    @ExceptionHandler(StationNotExistException.class)
+    public ResponseEntity<String> stationNotExistExceptionHandler(
+        final StationNotExistException exception) {
+        return ResponseEntity.badRequest()
+            .body(exception.getMessage());
+    }
```

Comment:
```suggestion
    @ExceptionHandler({SingleSectionException.class, StationNotExistException.class})
    public ResponseEntity<String> badRequestHandler(Exception exception) {
        return ResponseEntity.badRequest()
            .body(exception.getMessage());
    }
```

처리할 내용이 같으면 굳이 메소드를 분리할 필요는 없지 않을까요?
이 위에도 있는데 계속 추가되는 느낌이라서요~

Reply:
이렇게도 되는지 몰랐네요 :) 
알려주셔서 감사합니다 :)